packages/web/src/index.css

```diff
@@ -394,19 +394,19 @@
 }
 
 .cm-tooltip {
-  background-color: hsl(var(--background)) !important;
-  border-color: hsl(var(--border)) !important;
-  border-width: 1px;
-  border-radius: 0.1875rem;
-  overflow: hidden;
-  box-shadow:
-    0 4px 6px -1px rgb(0 0 0 / 0.1),
-    0 2px 4px -2px rgb(0 0 0 / 0.1);
+  @apply overflow-hidden;
+  @apply !bg-background;
+  @apply border;
+  @apply rounded-sm;
+  @apply shadow-md;
```

Comment:
Just a warning this will break if there are no other shadow-md uses in the code at some point since tailwind won't make the class

Reply:
I think it should be scanning our css files but admittedly haven't verified. `.css` is included in [our tailwind config](https://github.com/srcbookdev/srcbook/blob/main/packages/web/tailwind.config.js#L6) though.

Reply:
good question, I've never seen it used inside of the CSS file itself, worse case it breaks we search for the issue and the note will be here